jest.config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "testMatch": ["<rootDir>/tests/integration/reporter.spec.js"],
```

Comment:
Should this be a glob in case tests are added in the future?

Reply:
updated.